src/device-api/v2.ts

```diff
@@ -291,10 +292,14 @@ export function createV2Api(router: Router, applications: ApplicationManager) {
 			});
 		}
 
+		// FIXME: We need to validate the target state here,
+		// before emitting it
+
 		// Now attempt to set the state
 		const force = req.body.force;
 		const targetState = req.body;
 		try {
+			globalEventBus.getInstance().emit('targetStateChanged', targetState);
 			await deviceState.setTarget(targetState, true);
```

Comment:
Shouldn't the `deviceState.setTarget` do the emit? It seems like the more sensible place to put it because then it'll handle it for all callsites

Reply:
Nice, good point.